### PR TITLE
Add ImageFilter for image url to img tag conversion

### PR DESCRIPTION
**Pull request by st0012**
This simple filter converts url like: `https://example.com/test.png` into image tag like
```html
<img src="https://example.com/test.png" alt=""/>
```

### CODE REVIEW

File: lib/html/pipeline/image_filter.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module HTML
+  class Pipeline
+    # HTML Filter that converts image's url into <img> tag.
+    # For example, it will convert
+    #   http://example.com/test.jpg
+    # into
+    #   <img src="http://example.com/test.jpg" alt=""/>.
+
+    class ImageFilter < TextFilter
```
Comment (jch) [inline lib/html/pipeline/image_filter.rb]: Why inherit from `TextFilter` here?

**Comment by jch** [comment]
:+1: seems like a useful general filter. Once you address the inheritance comment, we should be good to merge this.

File: lib/html/pipeline/image_filter.rb (hunk 1 above)
Comment (st0012) [inline lib/html/pipeline/image_filter.rb]: Cause I want to use the `@text` variable to get the raw text content, and I want to make sure the content uses this filter is always string.

**Comment by jch** [comment]
Thanks!

**Pull request merged**